### PR TITLE
Improve DTS UI

### PR DESCRIPTION
**Pull request by DaniilKl**
Refers to issue https://github.com/Dasharo/dasharo-issues/issues/225.

### CODE REVIEW

**Comment by macpijan** [comment]
@DaniilKl I guess no new commits here yet?

File: scripts/ec_transition [hunk 1]
```diff
@@ -125,7 +125,7 @@ usage() {
 
 ROOT_DIR="/"
 
-[ -z "$BOARD_VENDOR" ] && error_exit "BOARD_VENDOR not given"
+[ -z "$SYSTEM_VENDOR" ] && error_exit "BOARD_VENDOR not given"
```
Comment (macpijan) [inline scripts/ec_transition:128]: Change the help text as well?

File: include/dts-functions.sh [hunk 2]
```diff
@@ -1212,3 +1224,134 @@ handle_fw_switching() {
...
+  else
+    echo -e "${BLUE}**${YELLOW}     ${DES_KEYS_OPT})${BLUE} Load your DES keys${NORMAL}"
+  fi
+  echo -e "${BLUE}*********************************************************${NORMAL}"
+  echo -e "${YELLOW}Select a menu option or${NORMAL}"
+  echo -ne "${RED}${REBOOT_OPT_UP}${NORMAL} to reboot  ${NORMAL}"
+  echo -ne "${RED}${POWEROFF_OPT_UP}${NORMAL} to poweroff  ${NORMAL}"
+  echo -e "${RED}${SHELL_OPT_UP}${NORMAL} to enter shell  ${NORMAL}"
+  if systemctl is-active sshd.socket &> /dev/null; then
+    echo -ne "${RED}${SSH_OPT_UP}${NORMAL} to stop SHH server  ${NORMAL}"
```
Comment (macpijan) [inline include/dts-functions.sh:1353]:
```suggestion
    echo -ne "${RED}${SSH_OPT_UP}${NORMAL} to stop SSH server  ${NORMAL}"
```

File: include/dts-functions.sh [hunk 3]
```diff
@@ -1212,3 +1224,134 @@ handle_fw_switching() {
...
+  else
+    echo -ne "${RED}${SSH_OPT_UP}${NORMAL} to launch SHH server  ${NORMAL}"
```
Comment (macpijan) [inline include/dts-functions.sh:1355]:
```suggestion
    echo -ne "${RED}${SSH_OPT_UP}${NORMAL} to launch SSH server  ${NORMAL}"
```

File: include/dts-environment.sh [hunk 4]
```diff
@@ -4,16 +4,32 @@
...
-BOARD_VENDOR="$(dmidecode -s system-manufacturer)"
+BLUE='\033[0;36m'
+
+# DTS options:
+HCL_REPORT_OPT="1"
+DASHARO_FIRM_OPT="2"
+REST_FIRM_OPT="3"
+DES_KEYS_OPT="4"
+REBOOT_OPT_UP="R"
+REBOOT_OPT_LOW="r"
```
Comment (macpijan) [inline include/dts-environment.sh:20]: If we have to define upper and lower, we can also convert one to another: https://stackoverflow.com/questions/2264428/how-to-convert-a-string-to-lower-case-in-bash

File: scripts/ec_transition (hunk 1 above)
Comment (DaniilKl) [reply scripts/ec_transition]: Amended here f921869fc12670429fe1ec65f81eb9b3253b9462.

File: include/dts-functions.sh (hunk 2 above)
Comment (DaniilKl) [reply include/dts-functions.sh]: Amended here dafce5b715af1efd86fd0e57035093b0188ee074.

File: include/dts-functions.sh (hunk 3 above)
Comment (DaniilKl) [reply include/dts-functions.sh]: Amended here dafce5b715af1efd86fd0e57035093b0188ee074

**Pull request merged**